frontend/app/components/work-packages/wp-attachments-formattable-field/wp-attachments-formattable.directive.ts

```diff
@@ -139,7 +191,7 @@ function wpAttachmentsFormattable() {
       if (angular.isUndefined(controllers[0] && angular.isUndefined(controllers[1]))) {
         return;
       }
-
+      console.log(controllers);
```

Comment:
Superfluous log line. It's the only remark I have on the PR, so I'll remove it after merging.